lib/plugin.js

```diff
@@ -1,13 +1,18 @@
 'use strict'
 
 const createError = require('http-errors')
+const extractSignatureHeaderAsDict = require('./extractSignatureHeaderAsDict')
 
 function plugin (pluginOptions) {
   const defaultOptions = {
     sharedSecret: null,
     verificationError: 'Unauthorized',
     verificationErrorMessage: 'Signature verification failed',
-    extractSignature: () => {},
+    extractSignature: req => {
```

Comment:
I think it would make sense for this method to live in its own file (`extractSignature.js`), that way we can easily test it as a standalone piece easily. It also seems like a natural seam for splitting this up. We can simply import extractSignature and pass that into pluginOptions.

Reply:
Should make testing more straightforward as well. I'll break this out.